### ISSUE TITLE
Firefox flatpak version not showing due to folder access permission.

### ISSUE THREAD

**Issue by queeup** [flathub/org.bleachbit.BleachBit#16]
Bleachbit need to access `~/.var/app` folder to clean Firefox flatpak version.

Please add `--filesystem=~/.var/app` to `finish-args`

**Comment by queeup** [comment]
@proletarius101, can you check this please? Without this permission BleachBit is not working with flatpak apps.

**Comment by bbhtt** [comment]
Should be fine do add if necessary. Does it actually work with that permission?

**Comment by bbhtt** [comment]
And what does it actually delete? I assume the `cache` folder? There's always the risk that these programs starts deleting the wrong things.

**Comment by queeup** [comment]
@bbhtt, Yes with that permission BleachBit recognize I am installed Firefox flatpak and clean it. Clean what ever you chose from the gui.

**Comment by queeup** [comment]
> @queeup I see your point. But I'm not sure if it's allowed by Flathub.

@proletarius101 Looks like it is: https://github.com/flathub/org.gnome.DejaDup/blob/04cf967b97d383f581ff7f4129a2f497ab117bb5/org.gnome.DejaDup.yaml#L16
